src/components/ec-submenu/ec-submenu.vue

```diff
@@ -2,26 +2,31 @@
   <div
     v-if="submenu && submenu.length > 0"
     class="ec-submenu"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-submenu` : 'ec-submenu'"
   >
     <div class="ec-submenu__header-container">
       <ul class="ec-submenu__header">
         <li
           v-for="(menuItem, index) in submenu"
           :key="index"
-          :data-test="'ec-submenu__header-item-' + index"
```

Comment:
I guess this remove of the index it's because we want to check better the title instead of the item right?

Reply:
I moved the `@click` events from li to anchor because of a11y and I wanted to move the data-test too, because usually we want to target in tests an element which has a click handler. I still left a data-test here, but it's not that important for us.

Reply:
Cool! thx for the explanation!